types/loadable__server/loadable__server-tests.tsx

```diff
@@ -34,7 +34,7 @@ const {
 // getLinkTags
 {
     // Should return an arary of strings
```

Comment:
Actually, while you're at it, test descriptions could be corrected to match the test here
```suggestion
    // Should return string
```